backend/module-campaigns/src/Podkrepibg.Campaigns.Infrastructure/Configuration/DbConfiguration.cs

```diff
@@ -0,0 +1,15 @@
+namespace Podkrepibg.Campaigns.Infrastructure.Configuration
+{
+    using System;
+
+    public class DbConfiguration
+    {
+        private static string DbHost = Environment.GetEnvironmentVariable("DB_HOST");
+        private static string DbPort = Environment.GetEnvironmentVariable("DB_PORT");
+
+        public static string GetConnectionString()
+        {
+            return $"Host={DbHost};Port={DbPort};Database=app;Username=root;Password=1234";
```

Comment:
Can you also make the name and password configurable

Reply:
In my opinion everything should be configurable. Even the user will be changing for the actual deployment

Reply:
Now all the strings are read from env